### PR TITLE
NEWS.md: efa and core update for 1.16.0

### PR DESCRIPTION
**Pull request by wzamazon**
Added news for EFA.

Also added 1 item for core regarding CUDA IPC support

Signed-off-by: Wei Zhang <wzam@amazon.com>

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -10,8 +10,14 @@ v1.16.0, Fri Sep 30, 2022
 =========================
 
 ## Core
+- Added CUDA IPC support
```
Comment (shijin-aws) [inline NEWS.md:13]: I would rephrase it to `Added HMEM IPC cache mechanism`. CUDA IPC support is already introduced in earlier libfabric versions.

Comment (wzamazon) [reply NEWS.md:13]: >CUDA IPC support is already introduced in earlier libfabric versions.

I am not sure. Previous version had ZE IPC, and SAR protocol for CUDA, but does not have CUDA IPC

But the item "CUDA IPC support" should be in SHM provider's NEWs, not core. Core should have "Added HMEM IPC cache mechanism" as you suggested.

Comment (wzamazon) [reply NEWS.md]: Looks like the shm change has already been covered in @aingerson's PR. so I did not include it in this PR

File: NEWS.md [hunk 2]
```diff
@@ -10,8 +10,14 @@ v1.16.0, Fri Sep 30, 2022
 =========================
 
 ## Core
+- Added HMEM IPC  cache
```
Comment (shijin-aws) [inline NEWS.md:13]: extra space between IPC and cache?

Comment (wzamazon) [reply NEWS.md]: Thank you! fixed

**Review by shijin-aws — APPROVED** [review]

**Comment by shefty** [comment]
Are you going to pick for v1.16.x?

**Comment by wzamazon** [comment]
Yes. opened backport PR https://github.com/ofiwg/libfabric/pull/8037
